content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md

```diff
@@ -162,7 +162,7 @@ To configure an apex domain, you only need to pick a single DNS record type from
 | Apex domain<br />(`example.com`) | `A` | `@` | `185.199.108.153`<br />`185.199.109.153`<br />`185.199.110.153`<br />`185.199.111.153` |
 | Apex domain<br />(`example.com`) | `AAAA` | `@` | `2606:50c0:8000::153`<br />`2606:50c0:8001::153`<br />`2606:50c0:8002::153`<br />`2606:50c0:8003::153` |
 | Apex domain<br />(`example.com`) | `ALIAS` or `ANAME` | `@` | `USERNAME.github.io` or<br /> `ORGANIZATION.github.io` |
-| Subdomain<br />(`ww​w.example.com`,<br />`blog.example.com`) | `CNAME` | `SUBDOMAIN` | `USERNAME.github.io` or<br /> `ORGANIZATION.github.io` |
+| Subdomain<br />(`ww​w.example.com`,<br />`blog.example.com`) | `CNAME` | `SUBDOMAIN.example.com.` | `USERNAME.github.io` or<br /> `ORGANIZATION.github.io` |
```

Comment:
FYI this was recently removed in #31358 — so just a heads up for triage to hopefully settle on a style and avoid changing it back and forth…

_(TBH: I'd personally like to see it added back like in this PR, though…)_

Reply:
@janbrasna Thank you for the reminder @janbrasna! 💛 Let me put this up for review so we can get a second opinion on this

CC @BenjaminBrienen just for visibility (thanks for your patience!)

Reply:
I'm sure a little extra review on this isn't blocking anyone 😁

Reply:
For the record… one half of the above mentioned PR got reverted pretty soon (something I didn't like back then, too) so this would revert the remaining change — and I see it as a good thing — the consistency callout with that specific page/table is my exact point from few months ago, too;)

And as DNS management UIs differ between providers there's not exactly one "correct" format for it per se to be completely copy-paste-ready all the time, so it's not worth fiddling with it too much beyond some basic correctness and consistency across the pages (in the future).

Reply:
Thanks for bearing with us (me) @janbrasna 💛 we'll hopefully have a concrete direction on this soon to be abide by for the foreseeable future (outside of basic correctness and consistency like you mentioned)

Reply:
Thanks so much @janbrasna and apologies for the confusion here—I'm not honestly sure what's happened but good that we're sorting it now.

> And as DNS management UIs differ between providers there's not exactly one "correct" format for it per se to be completely copy-paste-ready all the time, so it's not worth fiddling with it too much beyond some basic correctness and consistency across the pages (in the future).

Agree with you completely here, so I'm happy to go ahead and make this change after the freeze Alex mentioned. Thanks for picking this up, @BenjaminBrienen—we'll merge this next week 👍 

cc @nguyenalex836